File: CRUDAPI/Models/SignUp.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CRUDAPI.Models
{
    public class SignUp
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Gender { get; set; }
        public string Mobile { get; set; }
        public string Address { get; set; }
        public bool isActive { get; set; }
    }
}

File: CRUDAPPLICATION/wwwroot/js/SignUp.js
$(document).ready(function () {

    $("#btnSignUp").click(function () {
        var isValid = true
        isValid = requiredTextField("Name", " your name")
        if (!isValid) { return isValid }

        isValid = requiredTextField("Email", " your email address")
        if (!isValid) { return isValid }

        isValid = requiredTextField("Password", " your password")
        if (!isValid) { return isValid }

        isValid = requiredSelectField("Gender", " gender")
        if (!isValid) { return isValid }

        isValid = requiredTextField("Mobile", " your mobile number")
        if (!isValid) { return isValid }

        isValid = requiredTextField("Address", " your address")
        if (!isValid) { return isValid }

        console.log(isValid)
    })
})